gcf/lib/common.go

```diff
@@ -86,9 +86,17 @@ func WriteToGCS(ctx context.Context, path, data string) error {
 		return errors.Wrap(err, "Failed to create gcsClient")
 	}
 	w := gcsClient.Bucket(bucket).Object(object).NewWriter(ctx)
-	defer w.Close()
-	_, err = fmt.Fprint(w, data)
-	return errors.WithMessagef(err, "Failed to write data to %s/%s", bucket, object)
+	_, writeErr := fmt.Fprint(w, data)
+	closeErr := w.Close()
```

Comment:
Move this to after line 94?

Reply:
maybe need to close here even though write has error?